shared/src/test/scala/mlscript/DiffTests.scala

```diff
@@ -382,17 +434,22 @@ class DiffTests extends org.scalatest.funsuite.AnyFunSuite with org.scalatest.Pa
                       prefixLength = 3
                     }
                 }
-                showResult(prefixLength)
+                showFirstResult(prefixLength)
             }
 
             // If code generation fails, show the error message.
             results match {
-              case L((message, crashed)) =>
-                if (crashed)
-                  output("Code generation crashed:")
-                else
-                  output("Code generation met an error:")
-                output(s"  $message")
+              case IllFormedCode(message) =>
+                totalRuntimeErrors += 1
+                output("Code generation met an error:")
```

Comment:
`met` is a little strange here
```suggestion
                output("Code generation encountered an error:")
```